### PR TITLE
feat: add image and input components

### PR DESCRIPTION
**Pull request by irfan-nawaz**
- merging image and input components.
- refactor code of other components.
- add CSS variables for box-shadows

### CODE REVIEW

File: Components/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+/* Default validator for input*/
+.default-input input:focus{
+    border-bottom : 2px solid var(--primary-color);
+}
+
+/* Error validator for input*/
+.error-input input:focus{
+    border-bottom : 2px solid var(--error-color);
+}
+
```
Comment (starver20) [inline Components/Input/input.css]: Error effect color should remain all the time, not only on focus

**Review by starver20 — APPROVED** [review]
All good except the suggestion.👍🏻

**Review by shaik-saad — APPROVED** [review]
At some places you have used hyphen (-) and most of the places you've used under score (_) in the naming convention. Try to use only one of these as Pranshu has mentioned earlier about it. Other than that every thing looks good. You can merge.

File: Components/Input/input.css (hunk 1 above)
Comment (irfan-nawaz) [reply Components/Input/input.css]: done 🙌🏼

> At some places you have used hyphen (-) and most of the places you've used under score (_) in the naming convention. Try to use only one of these as Pranshu has mentioned earlier about it. Other than that every thing looks good. You can merge.

thanks, guys for reminding me I have done the suggested changes

**Pull request merged**